src/types.ts

```diff
@@ -1713,6 +1713,10 @@ export type MessageFilters<StreamChatGenerics extends ExtendableGenerics = Defau
     }
 >;
 
+export type MessageOptions = {
```

Comment:
I think that only `MessageOptions` would not be clear enough to convey the fact that these are actually search payload options.

```suggestion
export type SearchMessageOptions = {
```

Reply:
Yeah I though about it as well, however, technically, in the future, it could be used in other places where the `MessageFilters` is also used.